### ISSUE TITLE
New pages not getting created in public folder

### ISSUE THREAD

**Issue by pssguy** [rstudio/blogdown#19]
I'm creating a new RMarkdown file, eplBook,Rmd in the content/post folder

```
> serve_site()
```

> Error: Error copying static files to C:\Users\Andrew\Documents\R\mainWebSite\public: open C:\Users\Andrew\Documents\R\mainWebSite\public\img\Thumbs.db: Access is denied.

an html file is created in content/post 

![content](https://cloud.githubusercontent.com/assets/2775918/20724810/8fc11a36-b624-11e6-933a-a7ddad99f636.png)

but there is no related directory in public/post

![public](https://cloud.githubusercontent.com/assets/2775918/20724816/95b7ea50-b624-11e6-930c-95b081ad6012.png)

**Comment by yihui** [comment]
Sorry, I cannot help without a reproducible example, but at least you should ignore `Thumbs.db` by adding it to `ignoreFiles` in `config.toml`:

```
ignoreFiles = [ "^Thumbs\\.db$" ]
```

**Comment by pssguy** [comment]
Incorporating your suggestion made no difference. i.e error still occurs and folders and files do not get created under public/post

Not sure how to provide a reproducible example. Here is my eplBook.Rmd file

![probpublishing](https://cloud.githubusercontent.com/assets/2775918/21578785/0fb727e0-cf42-11e6-831c-601afcf73bf7.PNG)

Until there is a solution. can I just manually create the eplBook folder and files directories and add the created html and any figs to them??

BTW as per image, how would I add a post ot more than one category e.g 'books' and 'dashboards'

**Comment by yihui** [comment]
To provide a reproducible example, you can zip the website and upload here, or create a Github repo. The `public` should not be edited manually. Changes could be overwritten in the future.

For more categories, use `["books", "dashboards"]`. That is YAML syntax. More examples here: https://bookdown.org/yihui/bookdown/r-markdown.html

**Comment by pssguy** [comment]
@yihui 
Thanks for this. Here is the zipped file of the site
[mainWebSite.zip](https://github.com/rstudio/blogdown/files/685408/mainWebSite.zip)

Everything works smoothly up to and inc 27th Nov post.  so eplBook (which has just a title) and exploratoryAnalysis (which includes a notebook) do not appear in public folder when I serve_site(). Incidentally, I have to do latter. I had thought just saving the file activated this or there was a build buttton in RStudio to do it but I could well be mistaken

Also I set up git locally but cannot commit to it. There is (No branch) instead of the usual Master when i I view  the review changes.  Could this be part of the same problem?

**Comment by yihui** [comment]
I'm not sure if it is a Hugo bug, but I'd suggest you delete the Thumbs.db files. First take a look at

```r
list.files('.', '^Thumbs[.]db$', recursive = T, full.names = T)
```

If none of them seems to be useful (typically they are just Windows garbage), you can `unlink()` them.

I compiled your website on Mac and it works fine for me:

![image](https://cloud.githubusercontent.com/assets/163582/21670610/15104e3c-d2db-11e6-89b2-355a2f029713.png)

**Comment by pssguy** [comment]
Thanks for that. I also changed a setting in Windows so hopefully they do  recur

Incidentally, did you actually view the website in  Mac.  The exploratoryAnalysis page shows as a simple page for me i.e no Hugo stuff wrapping it. This happened even when I removed the output: html_notebook
I had experimented with
If you get same error I can raise as separate issue

**Comment by yihui** [comment]
The exploratory analysis page looks okay to me:

![image](https://cloud.githubusercontent.com/assets/163582/21696106/1d4a1cc8-d352-11e6-9e49-b0eaf0f14017.png)

**Comment by yihui** [comment]
I feel I might have misled you somehow by having picked up a fairly complicated Hugo theme (hugo-future-imperfect) as the default in `new_site()`. Now I have changed the default and used a much simpler theme ([hugo-lithium-theme](https://github.com/yihui/hugo-lithium-theme)). You can still use hugo-future-imperfect, but I think you will have to read the Hugo documentation a few times as well as the blogdown documentation, which does not exist yet (I'm working on it).

**Comment by pssguy** [comment]
FYI an update
I had a look at the index.html file  for that post and it had 
<meta name="generator" content="pandoc" /> and was extremely large. All the others were
<meta name="generator" content="Hugo 0.17" />

I updated to 0.0.11 (from 0.0.8) and got rid of the nb.html file that was around  and for a while that seemed to solve issue.  
However, as soon as I started attempting any changes and/or getting rid of unwanted posts I ended up with only three of seven posts appearing and a config.toml file of 6.8kb mainly made of whitespace.  I will take your advice and use a simpler theme at least for now and see if that sorts my problems
